src/main/java/com/iota/iri/storage/Tangle.java

```diff
@@ -369,4 +374,15 @@ public void clearMetadata(Class<?> column) throws Exception {
             provider.clearMetadata(column);
         }
     }
+
+    public <T> Cache<Indexable, T> getCache(Class<T> type){
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Public method and constructor comments are required](https://app.codacy.com/app/GalRogozinski/iri/pullRequest?prid=4567264)